$OpenBSD: patch-configure,v 1.9 2013/06/17 19:30:22 ajacoutot Exp $

XXX patch evo_krb5_support.m4 to use krb5-config(1) and push upstream

--- configure.orig	Sat Jun  8 19:54:10 2013
+++ configure	Thu Jun 13 17:32:26 2013
@@ -22114,7 +22114,7 @@ $as_echo_n "checking for Kerberos 5... " >&6; }
 		esac
 
 		mitlibs="-lkrb5 -lk5crypto -lcom_err -lgssapi_krb5"
-		heimlibs="-lkrb5 -lcrypto -lasn1 -lcom_err -lroken -lgssapi"
+		heimlibs="`krb5-config --libs gssapi`"
 		sunlibs="-lkrb5 -lgss"
 		if ${ac_cv_lib_kerberos5+:} false; then :
   $as_echo_n "(cached) " >&6
@@ -22226,7 +22226,7 @@ $as_echo "#define HAVE_HEIMDAL_KRB5 1" >>confdefs.h
 					if test -z "$with_krb5_includes"; then
 						KRB5_CFLAGS="-I$with_krb5/include/heimdal"
 					else
-						KRB5_CFLAGS="-I$with_krb5_includes"
+						KRB5_CFLAGS="`krb5-config --cflags`"
 					fi
 					msg_krb5="yes (Heimdal)"
 				else
